### DISCUSSION TITLE
Series Expansion Method terms

### DISCUSSION THREAD

**Discussion by jinhray** [mtex-toolbox/mtex#1889] [Ask Anything]
Dear all!

I understand that the method used by mTex in ODF reconstruction uses spherical harmonics series expansion. Now, I want to manually decide truncation level l_max used to calculate the value of the series expansions. 

In practical terms, a value of l_max = 22 is generally taken as a limit for cubic materials, whereas for very sharp textures, a value of l_max = 34  is considered to yield sufficiently accurate results. 

Since I want to produce ODFs for l_max = 8:2:34 for study, I want to decide on it.

I tried using FourierODF(odf, term); with term = 8:2:34 

However, maybe my approach is wrong? 

Hopefully you could help me! Thank you! 

JJ

**Comment by ralfHielscher** [comment]
Hi,

in order to specify the bandwidth use the syntax
```matlab
odf_harm = FourierODF(odf, 'bandwidth',8)
```
You should get something like
```
odf_harm = SO3FunHarmonic (Quartz → xyz)
  bandwidth: 8
  weight: 1
```
Ralf.

**Comment by jinhray** [comment]
Dear Mr. Ralf,

Thank you for the reply!

Is the bandwidth similar to the truncation level? From my understanding, it helps smooth the ODF but fundamentally, they should be different.

Just to be more precise, I mean to modify the spherical harmonic degree. Since the ODF function is defined with respect to both Fourier coefficients and the spherical harmonic of degree l and order m. If we can modify Fourier coefficients, can we decide on l?

Thank you!

JJ

**Reply by ralfHielscher** [reply]
The bandwidth and the truncation level describe the same thing. 
You may do
```matlab
plotSpektra(odf_harm) 
```
to visualize the decay of the harmonic coefficients. 
Ralf.
